samples/simple_app/app.py

```diff
@@ -61,34 +51,55 @@ async def get_webhook_details(domain_name: DomainName) -> WebhookData:
         webhook_secret_key="webhook-secret-key",
     )
 
+async def example_dependency():
+    return "example"
+
 
-async def product_created(payload: Payload, saleor_domain: DomainName):
+async def product_created(payload: List[Webhook], saleor_domain=Depends(saleor_domain_header), example = Depends(example_dependency)):
+    print(example)
+    print(saleor_domain)
     print("Product created!")
     print(payload)
 
 
-async def product_updated(payload: Payload, saleor_domain: DomainName):
+async def product_updated(payload: List[Webhook], saleor_domain=Depends(saleor_domain_header), example = Depends(example_dependency)):
+    print(example)
+    print(saleor_domain)
     print("Product updated!")
     print(payload)
 
 
-async def product_deleted(payload: Payload, saleor_domain: DomainName):
+async def product_deleted(payload: List[Webhook], saleor_domain=Depends(saleor_domain_header), example = Depends(example_dependency)):
+    print(example)
+    print(saleor_domain)
     print("Product deleted!")
     print(payload)
 
 
-webhook_handlers = WebhookHandlers(
-    product_created=product_created,
-    product_updated=product_updated,
-    product_deleted=product_deleted,
-)
-
-
 app = SaleorApp(
+    manifest=Manifest(
```

Comment:
Settings is something that defines how an application changes from deployment stage to deployment stage, i.e. prod vs test, the Manifest should not change like that, not unless we consider URLs but there's no need for that thanks to `url_for`. 
With this, the manifest describes the app, which is a better description of what it's really doing